app/Plugin/CoreApi/Lib/lang.php

```diff
@@ -41,17 +41,25 @@
     CoreApiEnum::CoPersonRead     => 'COmanage CO Person Read API',
     CoreApiEnum::CoPersonWrite    => 'COmanage CO Person Write API'
   ),
-  
+
+  'pl.coreapi.en.response.type' => array(
+    ResponseTypeEnum::Full              => 'Full Profile(s)',
```

Comment:
I'd wordsmith these to `Full Profile` and `Identifier Only`.

Reply:
Done